health/micro-ui/web/micro-ui-internals/packages/modules/pgr/src/hooks/pgr/useServiceDefs.js

```
@@ -5,17 +5,16 @@ const { useState, useEffect } = require("react");
 const useServiceDefs = (tenantId, moduleCode) => {
   const [localMenu, setLocalMenu] = useState([]);
   const SessionStorage = Digit.SessionStorage;
-  let { t } = useTranslation();
 
   useEffect(() => {
     (async () => {
       const serviceDefs = await Digit.MDMSService.getServiceDefs(tenantId, moduleCode);
       SessionStorage.set("serviceDefs", serviceDefs);
 
-      const serviceDefsWithKeys = serviceDefs.map((def) => ({ ...def, i18nKey: t("SERVICEDEFS." + def.serviceCode.toUpperCase()) }));
+      const serviceDefsWithKeys = serviceDefs.map((def) => ({ ...def, i18nKey: "SERVICEDEFS." + def.serviceCode.toUpperCase() + `.${def.department}`, code: `${def.serviceCode}.${def.department}` }));
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)